common/app/model/content/Atoms.scala

```diff
@@ -288,6 +288,50 @@ object InteractiveAtom {
 
 object RecipeAtom {
   def make(atom: AtomApiAtom): RecipeAtom = RecipeAtom(atom.id, atom, atom.data.asInstanceOf[AtomData.Recipe].recipe)
+
+  def getPicture(r: RecipeAtom, defaultImage: model.ImageMedia): model.ImageMedia = {
+    r.data.images.headOption.map{ img => MediaAtom.imageMediaMake(img, "caption here")}.getOrElse(defaultImage)
```

Comment:
what about naming this function `picture`?

I would keep the `getOrElse` logic out of this method and have it return an `Option[ImageMedia]`. Is it not really the concern of the `RecipeAtom` to handle what needs to be done when no picture